packages/tools/client-debugger/devtools/package.json

```
@@ -0,0 +1,96 @@
+{
+	"name": "@fluid-experimental/devtools",
+	"version": "2.0.0-internal.4.2.0",
+	"private": true,
+	"description": "Fluid Framework developer tools",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/tools/client-debugger/devtools"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
+		"build:compile": "npm run build:tsc",
```

There was a recent PR by Curtis that removed the `build:tsc` script. In those cases I think there's commonjs and esnext so `build:compile` runs them both concurrently, but here I think we can just do this (or `npm run tsc`)? (And remove `build:tsc` below).
```suggestion
		"build:compile": "tsc",
```

---

If I remember correctly, there are some places in our build tooling that look for a "tsc" script, so I went with `npm run tsc` for now. We can always further clean this later if we'd like to.